pkg/capture/capture_manager.go

```diff
@@ -227,6 +227,9 @@ func (cm *Manager) Status(ctx context.Context, ifaces ...string) (statusmap capt
 
 	statusmap = make(capturetypes.InterfaceStats)
 
+	cm.Lock()
```

Comment:
This was missing entirely, allowing two concurrent calls to `Status()` to run, e.g. from an ongoing scheduled rotation and one from an interface config change (nasty side-effects, including panics).